src/test/resources/basic-type-test/Request/BasicType.elm

```diff
@@ -0,0 +1,73 @@
+{-
+  This file was generated by endpoints-elm interpreter.
+  Do not edit this file manually.
+
+  See https://github.com/scalalandio/endpoints-elm for more information.
+-}
+
+module Request.BasicType exposing (..)
+
+import Http
+import HttpBuilder exposing (RequestBuilder)
+import Json.Decode as Decode
+import Bool.Extra
+import Maybe.Extra
+
+import Uuid exposing (..)
+
+
+unitechoGet : RequestBuilder ()
+unitechoGet  =
+  HttpBuilder.get "/unit-echo"
+    |> HttpBuilder.withExpect (Http.expectStringResponse (\_ -> Ok ()))
+    |> HttpBuilder.withTimeout 30000
+
+
+stringechoGet : String -> RequestBuilder String
+stringechoGet  =
+  HttpBuilder.get ("/string-echo/" ++ )
+    |> HttpBuilder.withExpectJson Decode.string
+    |> HttpBuilder.withTimeout 30000
+
+
+intechoGet : Int -> RequestBuilder Int
+intechoGet  =
+  HttpBuilder.get ("/int-echo/" ++ String.fromInt )
+    |> HttpBuilder.withExpectJson Decode.int
+    |> HttpBuilder.withTimeout 30000
+
+
+longechoGet : Int -> RequestBuilder Int
+longechoGet  =
+  HttpBuilder.get ("/long-echo/" ++ String.fromInt )
+    |> HttpBuilder.withExpectJson Decode.int
+    |> HttpBuilder.withTimeout 30000
+
+
+floatechoGet : String -> RequestBuilder Float
+floatechoGet  =
+  HttpBuilder.get ("/float-echo/" ++ )
```

Comment:
This is not valid elm code as you don't have segment instances for floats, doubles, and booleans. I guess in practice they are not so relevant, as usually, you don't pass such types as part of the query path, but rather as a query parameter (after `?`).

Anyway, we may have better error handling so that we won't produce invalid elm code.

Reply:
For now I removed that